pkg/config/config_test.go

```diff
@@ -23,9 +23,12 @@ func TestRead(t *testing.T) {
 			want: &FromFile{
 				SourceRepositories: []SourceRepository{
 					{
-						URL:          "https://github.com/giantswarm/apiextensions",
-						Organization: "giantswarm",
-						ShortName:    "apiextensions",
+						URL:             "https://github.com/giantswarm/apiextensions",
+						Organization:    "giantswarm",
+						ShortName:       "apiextensions",
+						AnnotationsPath: []string{"pkg/annotations"},
+						CRDPaths:        []string{"config/crd", "helm"},
```

Comment:
Everything in `config/crd` should be copied into `helm/*/templates` so it should be safe to delete `config/crd`.

Reply:
FYI: This test only verifies that we read the configuration file okay.